bin/bootstrap

```diff
@@ -17,6 +17,14 @@ if command -v salt &> /dev/null ; then
   brew uninstall nginx
   rm -rf $HOME/.lldev
 fi
+# Uninstall Dnsmasq
+# We new prefer configure /etc/hosts because
+# of port collision with Umbrella Roaming Client
+if command -v dnsmasq &> /dev/null ; then
```

Comment:
This did not uninstall dnsmasq for me. Interestingly, `command -v dnsmasq` does not return anything, so this code assumes it isn't there and does not uninstall it. However, `brew list` does show it as installed by brew.